nixos/tests/postgresql/postgresql.nix

```diff
@@ -26,6 +26,16 @@ let
     INSERT INTO sth (id) VALUES (1);
     CREATE TABLE xmltest ( doc xml );
     INSERT INTO xmltest (doc) VALUES ('<test>ok</test>'); -- check if libxml2 enabled
+    -- check if hardening gets relaxed
+    CREATE EXTENSION plv8;
+    -- try to trigger the V8 JIT, which requires MemoryDenyWriteExecute
+    DO $$
+      let xs = [];
+      for (let i = 0, n = 4000000000; i < n; i++) {
+          xs.push(Math.round(Math.random() * n))
+      }
+      console.log(Math.sum(xs));
```

Comment:
```suggestion
      let xs = [];
      for (let i = 0, n = 400000; i < n; i++) {
          xs.push(Math.round(Math.random() * n))
      }
      console.log(xs.reduce((acc, x) => acc + x, 0));
```

Reply:
Applied and tested that it actually triggers the JIT. Thank you!